utils/base64url/base64url.go

```diff
@@ -0,0 +1,36 @@
+package base64url
```

Comment:
Some reading: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Just place this next to jkws.

Yep, `utils` are there but will be removed in the future.